org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/OurSyntaxWidget.java

```diff
@@ -759,6 +768,11 @@ boolean saveAs(String filename, Collection<String> bannedNames) {
         }
         this.filename = Util.canon(filename); // a new file's canonical name may
                                              // change
+        try{
+            fileModifiedDate = Util.getModifiedDate(this.filename);
+        }catch(Exception e){
```

Comment:
nitpick: can you please add a space after each `}` and a space before each `{`, just to be consistent with the rest of the codebase?